tests/cover/test_recursive.py

```diff
@@ -153,6 +154,7 @@ def hard(base):
 
 @given(st.randoms(), settings=Settings(max_examples=10))
 @timeout(60)
+@flaky(max_runs=5, min_passes=1)
```

Comment:
Actually on reread... I think this should go outside the timeout, no? Otherwise the timeout will kill it before it can retry.